services/frontend-service/src/ui/components/ReleaseCard/ReleaseCard.tsx

```diff
@@ -15,11 +15,52 @@ along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>
 Copyright 2023 freiheit.com*/
 import classNames from 'classnames';
 import { Button } from '../button';
+import { Tooltip } from '../tooltip/tooltip';
 import React, { useEffect, useRef } from 'react';
 import { MDCRipple } from '@material/ripple';
 import { updateReleaseDialog, useRelease } from '../../utils/store';
 import { EnvironmentGroupChipList } from '../chip/EnvironmentGroupChip';
 
+const getRelativeDate = (date: Date): string => {
+    const millisecondsPerHour = 1000 * 60 * 60; // 1000ms * 60s * 60m
+    const elapsedTime = Date.now().valueOf() - date.valueOf();
+    const hoursSinceDate = Math.floor(elapsedTime / millisecondsPerHour);
+
+    if (hoursSinceDate < 24) {
+        // recent date, calculate relative time in hours
+        if (hoursSinceDate === 0) {
+            return '< 1 hour ago';
+        } else if (hoursSinceDate === 1) {
+            return '1 hour ago';
+        } else {
+            return `${hoursSinceDate} hours ago`;
+        }
+    } else {
+        // too many hours, calculate relative time in days
+        const daysSinceDate = Math.floor(hoursSinceDate / 24);
+        if (daysSinceDate === 1) {
+            return '1 day ago';
+        } else {
+            return `${daysSinceDate} days ago`;
+        }
+    }
+};
+
+export const getFormattedReleaseDate = (createdAt: Date): JSX.Element => {
+    // date format: dd-mm-yyyy, with no leading zeros, month is 0-indexed.
+    const formattedDate = `${createdAt.getDate()}-${createdAt.getMonth() + 1}-${createdAt.getFullYear()}`;
```

Comment:
I know this is like in the design, but I think we should change it.
I would much prefer an ISO-like date:
2023-12-31